storybook/stories/Layout/AppHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { AppHeader } from "../../../src/components/Layout/AppHeader";
+import { View, Text } from "react-native";
+import { size, color } from "../../../src/common/styles";
+import { AppMode } from "../../../src/context/config";
+import { createAppContainer } from "react-navigation";
+import { createDrawerNavigator } from "react-navigation-drawer";
+
+const reactNavigationDecorator = (story: any): any => {
+  const Screen = (): any => story();
+  const Navigator = createAppContainer(
+    createDrawerNavigator(
+      { Screen },
+      {
+        drawerPosition: "right",
+        drawerType: "slide",
+      }
+    )
+  );
+  return <Navigator />;
+};
+
+const appHeaderElements = (): any[] => {
+  return [
+    <>
+      <Text>Production App Header</Text>
+      <AppHeader />
+    </>,
+    <>
+      <Text>Staging App Header</Text>
+      <AppHeader mode={AppMode.staging} />
+    </>,
+  ];
+};
```

Comment:
Can we define this as an array of ReactElements instead? What's the reason for having a function that returns `any[]`?

This comment relates to all stories